lex_optimize3.py returned with 1

Traceback (most recent call last):
  ...
IOError: [Errno 2] No such file or directory: 'lexdir/sub/calctab.py'

